application/src/main/java/org/togetherjava/tjbot/commands/tags/TagSystem.java

```diff
@@ -54,19 +67,41 @@ static Button createDeleteButton(String componentId) {
      *
      * @param id the id of the tag to check
      * @param event the event to send messages with
+     * @param componentIdGenerator the component id generator to generate the button ids with
```

Comment:
what buttons? could help to give some context to the javadoc. such as `@param componentIdGenerator used to generate buttons with tag suggestions`